build.gradle

```diff
@@ -271,6 +271,7 @@ shadowJar {
     zip64 true
     dependencies {
         // exclude Hadoop dependencies, since they are provided when running with Spark
+        // TODO: improve this, since it doesn't take transitive dependencies into account
```

Comment:
Opened https://github.com/broadinstitute/hellbender/issues/836 and added a reference to it in the comment.